src/components/common.js

```diff
@@ -276,15 +276,26 @@ const commonSelectProps = {
     singleValue: base => ({ ...base, color: colors.dark() }),
     option: (base, { isSelected, isFocused, isDisabled }) => _.merge(base, {
       overflowWrap: 'break-word',
-      backgroundColor: isSelected ? colors.light(0.4) : isFocused ? colors.dark(0.15) : undefined,
-      color: isSelected ? colors.accent() : isDisabled ? undefined : colors.dark(),
+      fontWeight: isSelected ? 600 : undefined,
+      backgroundColor: isFocused ? colors.dark(0.15) : 'white',
```

Comment:
nit: I haven't seen 'white' as a color used much in the app, should this be 'colors.light' instead?

Reply:
+1 The hard coded color may make it more difficult to change the theme in the future. Especially if adding a high contrast theme for a11y - the hardcoded color may make this more difficult

Reply:
I see 66 uses of `white` vs just 27 uses of `light` in the code, I'm going to leave this as-is for now and we can consider a wider refactor later.

Reply:
I see 66 uses of `white` vs just 27 uses of `light` in the code, I'm going to leave this as-is for now and we can consider a wider refactor later.